Packs/GCP-Enrichment-Remediation/Playbooks/playbook-Cloud_Response_-_GCP.yml

```diff
@@ -1785,19 +1745,19 @@ tasks:
       '#default#':
       - "50"
       "yes":
-      - "49"
+      - "51"
     separatecontext: false
     conditions:
     - label: "yes"
       condition:
       - - operator: isEqualString
           left:
             value:
-              simple: identity_invoked_by_type
+              simple: inputs.accountType
             iscontext: true
           right:
             value:
-              simple: service_account
```

Comment:
Why did you change it?

Reply:
I thought it would be easier to use it without `_`, 'cause it will be controlled through the input. Do you see an issue using that in it's newer version?